### PR TITLE
Update URLs on Find and Publish to reflect new GIT Links

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Context

- These 2 URLs have changed

### Changes proposed in this pull request

- I've updated these 2 URLs

### Guidance to review

- Have the URLs updated?

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Inform data insights team due to database changes

### CODE REVIEW

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

File: config/locales/en.yml [hunk 1]
```diff
@@ -1019,8 +1019,9 @@ en:
   get_into_teaching:
     tel: 0800 389 2500
     opening_times: "Monday to Friday, 8.30am to 5.30pm"
-    url_get_an_advisor: https://adviser-getintoteaching.education.gov.uk
-    url_online_chat: https://getintoteaching.education.gov.uk/#talk-to-us
-    url_ways_to_train: https://getintoteaching.education.gov.uk/ways-to-train
+    url_get_an_advisor: https://getintoteaching.education.gov.uk/teacher-training-adviser/sign_up/identity
+    url_online_chat: https://getintoteaching.education.gov.uk/help-and-support
+    url_ways_to_train_no_degree: https://getintoteaching.education.gov.uk/train-to-be-a-teacher/if-you-dont-have-a-degree
```
Comment (MylesJarvis) [inline config/locales/en.yml:1024]: I've expanded the scope of the PR a bit, adding two new URLs to reflect some work on GIT to better target next steps for 'ways to train'.

File: app/views/find/courses/_advice.html.erb [hunk 2]
```diff
@@ -7,9 +7,8 @@
     <%= govuk_link_to "chat online", t("get_into_teaching.url_online_chat") %>
     <%= t("get_into_teaching.opening_times") %> (except public holidays).</p>
 
-  <p class="govuk-body">Find out about the different
-  <%= govuk_link_to("ways to train", t("get_into_teaching.url_ways_to_train")) %>
-  if you do not have a degree, or are already an experienced teacher.</p>
+  <p class="govuk-body">Find out about the different ways to train <%= govuk_link_to("if you do not have a degree", t("get_into_teaching.url_ways_to_train_no_degree")) %>
+  , or <%= govuk_link_to("are already an experienced teacher", t("get_into_teaching.url_ways_to_train_experienced_teacher")) %>.</p>
```
Comment (JR-G) [inline app/views/find/courses/_advice.html.erb:11]: There's an extra space here before the comma:
image

Comment (MylesJarvis) [reply app/views/find/courses/_advice.html.erb]: I've pushed a new commit for this

**Review by JR-G — APPROVED** [review]

**Pull request merged**